README.md

```diff
@@ -115,146 +60,67 @@ pub fn render() {
 }
 ```
 
-## Building and running
-
-To build your app, run `cargo make build`, and to host on a dev server, run `cargo make serve`.
-
-For a more robust starting setup, check out Martin Kavik's [seed-quickstart-webpack repo](https://github.com/seed-rs/seed-quickstart-webpack).
-
-## Running included examples
-
-To run an example located in the [examples folder](https://github.com/seed-rs/seed/tree/master/examples),
-run `cargo make start example_name`, where you replace `example_name` with the example name. Eg:
-`cargo make start counter`.
-
-Some examples also require to run API server in another terminal window - `cargo make start_server example_name`.
-
-When server(s) are running, open [127.0.0.1:8000](http://127.0.0.1:8000) in your browser.
-
-## Resources
+# Examples
+The examples provided in this repository are a good place to get started. They also act as an integration testing suite we run before every commit to ensure there are no unintended breaking changes to the user space. Because of Rust's compile-time checking, testing is that much more robust and changes can be pushed confidently.
+
+Run [examples](examples/) with `cargo make start example_name` from the Seed repository root.
+
+# Why Use Seed
+
+### Rust
+You may prefer writing in Rust and appreciate its benefits, including:
+- Rust **safety**.
+- Rust **compile-time error, type, and immutability checking**.
+- Rust built-in testing.
+- Rust speed.
+- Cleaner code and less runtime errors.
+- Cargo packages.
+
+### Development
```

Comment:
I would change it to something like `Seed's goals` or `Seed's focus`.., because as a reader I would expect details about Seed development (who is maintainer, how to write PR, Seed architecture explanation, etc.) instead of Seed's goals / focus / features / benefits.

Reply:
Or at least
```suggestion
### Development with Seed
```